### DISCUSSION TITLE
Bitbucket DataCenter and Jenkins Support

### DISCUSSION THREAD

**Discussion by jake6532** [diggerhq/digger#1160] [General]
Just curious, looking through the docs and the blog it's hard to tell if other CI/CD pipelines like Jenkins are currently supported or if that is just in the works? Also, on the pricing page it mentions if you want to use private VCS's that you need to purchase the enterprise edition.  Is Bitbucket DataCenter currently supported and if so, is that only as part of the enterprise edition?

**Comment by jake6532** [comment]
Also, maybe i missed it in the docs but are there REST endpoints for manually triggering plans and applies for integrating with other workflows that may require executions outside of PR's.

**Comment by UtpalJayNadiger** [comment]
Hi @jake6532 - yes BitBucket DataCenter is enterprise only. Jenkins support will be enabled on the Self Managed community edition shortly. 

> Are there REST endpoints for manually triggering plans and applies for integrating with other workflows

Not yet but would be great if you're able to create an issue. That'd help us enormously in scoping this, as we're also actively thinking about it.

Also please note most community discussions happen on our community slack: https://bit.ly/diggercommunity, so feel free to share any subsequent thoughts there, closing this discussion for now.
